crow-nado/chesspeak
# Capture logic

To test:
* Run rake db:migrate
* Confirm you cannot select pieces from both teams each turn
* Confirm pieces can't capture their same color
